docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+services:
+  db:
+    image: postgres:12
+    environment:
+      - POSTGRES_DB=${DATABASE_NAME:-navody_slovensko_digital_development}
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    expose:
+      - 5432/tcp
+    networks:
+      - navody
+    ports:
+      - 5432:5432
+    restart: unless-stopped
+    volumes:
+      - pg_data:/var/lib/postgresql/data
+  app:
+    image: navody
+    build: .
+    command: "tail -f /dev/null"
+    depends_on:
+      - db
+    environment:
+      - RAILS_ENV=development
+      - DATABASE_URL=postgresql://postgres@db/${DATABASE_NAME:-navody_slovensko_digital_development}
+    networks:
+      - navody
+    ports:
+      - 3000:3000
+    restart: unless-stopped
+    volumes:
+      - ./:/app
+      - bundler:/app/vendor/bundle
+      - node_modules:/app/node_modules
```

Comment:
preco potrebujeme tieto volumes takto?

Reply:
* `./:/app` - tento je kvoli developmentu, aby si mohol robit pripadny autoreload appky
* tie ostatne 2 volumes, uplne nutne netreba, kedze ak by sme ich aj nedefinovali, gemy resp. node moduly sa nainstaluju na hostitelsky fs. No zvyknem to pouzivat, ak na projekte pracuju aj MacOS developery, kde mali/maju tieto host fs bind-mounty [performance problemy](https://docs.docker.com/docker-for-mac/osxfs-caching/#performance-implications-of-host-container-file-system-consistency) - kedze pouzivam Linux, neviem checknut, aky velky to moze byt rozdiel na tomto projekte = treba skusit a pripadne mozeme tieto vymazat/dalej potweakovat :-)